src/rsz/src/Resizer.cc

```diff
@@ -762,35 +904,82 @@ bool Resizer::isTristateDriver(const Pin* pin)
 void Resizer::bufferOutput(const Pin* top_pin, LibertyCell* buffer_cell)
 {
   NetworkEdit* network = networkEdit();
-  Term* term = network_->term(top_pin);
-  Net* output_net = network_->net(term);
+
+  odb::dbITerm* top_pin_op_iterm;
+  odb::dbBTerm* top_pin_op_bterm;
+  odb::dbModITerm* top_pin_op_moditerm;
+  odb::dbModBTerm* top_pin_op_modbterm;
+
+  db_network_->staToDb(top_pin,
+                       top_pin_op_iterm,
+                       top_pin_op_bterm,
+                       top_pin_op_moditerm,
+                       top_pin_op_modbterm);
+
+  //  Term* term = network_->term(top_pin);
```

Comment:
rm // line

Reply:
Removed. Note how top pins are (weirdly) called Terms in the original data model and what we have done with the lines preceeding it is cover the top_pin to something we understand. The original code used Term but we have converted it to a db type.